src/SqlAsyncCollector.cs

```diff
@@ -504,6 +507,7 @@ public static async Task<TableInformation> RetrieveTableInformationAsync(SqlConn
             {
                 Dictionary<TelemetryPropertyName, string> sqlConnProps = sqlConnection.AsConnectionProps();
                 TelemetryInstance.TrackEvent(TelemetryEventName.GetTableInfoStart, sqlConnProps);
+                logger.LogDebugWithThreadId("BEGIN RetrieveTableInformationAsync");
```

Comment:
We should log all queries too so we can see exactly what is being executed in case there's some edge case we aren't handling correctly